frontend/services/entities/mdm.ts

```diff
@@ -324,13 +338,51 @@ const mdmService = {
     )}`;
     return sendRequest("DELETE", path);
   },
+
   getCommandResults: (
     command_uuid: string
   ): Promise<IGetMdmCommandResultsResponse> => {
     const { COMMANDS_RESULTS: MDM_COMMANDS_RESULTS } = endpoints;
     const url = `${MDM_COMMANDS_RESULTS}?command_uuid=${command_uuid}`;
     return sendRequest("GET", url);
   },
+
+  getSetupExperienceScript: (
+    teamId: number
+  ): Promise<IGetSetupExperienceScriptResponse> => {
+    const { MDM_SETUP_EXPERIENCE_SCRIPT } = endpoints;
+
+    const path = `${MDM_SETUP_EXPERIENCE_SCRIPT}?${buildQueryStringFromParams({
+      team_id: teamId,
+    })}`;
+
+    return new Promise((resolve, reject) => {
+      resolve(createMockSetupExperienceScript());
+    });
+
+    // return sendRequest("GET", path);
+  },
+
+  downloadSetupExperienceScript: (teamId: number) => {
+    const { MDM_SETUP_EXPERIENCE_SCRIPT } = endpoints;
+
+    const path = `${MDM_SETUP_EXPERIENCE_SCRIPT}?${buildQueryStringFromParams({
+      team_id: teamId,
+      alt: "media",
+    })}`;
+
+    return sendRequest("GET", path);
+  },
+
+  uploadSetupExperienceScript: (file: File, teamId: number) => {
+    const { MDM_SETUP_EXPERIENCE_SCRIPT } = endpoints;
+
+    const formData = new FormData();
+    formData.append("script", file);
+    formData.append("team_id", teamId.toString());
+
+    return sendRequest("POST", MDM_SETUP_EXPERIENCE_SCRIPT, formData);
+  },
 };
 
```

Comment:
Should this also include `deleteSetupExperienceScript`? 

On the backend, no team is implemented as `DELETE /setup_experience/script` and teams are implemented as `DELETE /setup_experience/script?team_id=1`

Reply:
yes we should I will do that in the integration PR. thanks for the heads up